### ISSUE TITLE
Support Ruby 3.0 / IRB 1.2.6

### ISSUE THREAD

**Issue by fig** [amazing-print/amazing_print#56]
IRB v1.2.6 introduced an option to omit / truncate overly long lines of output on assignment. https://github.com/ruby/irb/commit/c5ea79d5cecea9cae6ad0c1f31703a98cd329431#diff-ed2717fee59c6f4da595a7fe4ed65be8R749

The implementation details include passing an optional argument `[omit]` to `IRB::Irb#output_value`.

```ruby
# lib/irb.rb

module IRB
  class Irb
  ...
    def output_value(omit = false)
      ...
    end
  ...
  end
end
```

AmazingPrint redefines `output_value` without arguments and so raises `ArgumentError: wrong number of arguments (given 1, expected 0)` when used with IRB >= 1.2.6. (installed with Ruby 3.0)

```ruby
# lib/amazing_print/custom_defaults.rb

module AmazingPrint
  class << self
  ...
    def usual_rb
      IRB::Irb.class_eval do
        def output_value
          ap @context.last_value
        rescue NoMethodError
          puts "(Object doesn't support #ai)"
        end
      end
    end
    ...
  end
end
```

I've done some brief experimenting with adding an unused optional method argument to the method signature of `output_value` and so far, it appears to fix the compatibility issue without side effects.

**Comment by mattbrictson** [comment]
This seems to be causing problems in Ruby 2.7.2 as well, which was just released. Ruby 2.7.2 is packaged with IRB 1.2.6.

**Comment by paddor** [comment]
> This seems to be causing problems in Ruby 2.7.2 as well, which was just released. Ruby 2.7.2 is packaged with IRB 1.2.6.

Does it work with the merged fix?

**Comment by mattbrictson** [comment]
Yes, I just verified that merged #57 fixes the error in Ruby 2.7.2 as well. Thanks!
